diff --git a/setup.py b/setup.py
index e3590d2..905105d 100644
--- a/setup.py
+++ b/setup.py
@@ -56,9 +56,10 @@ class PostInstallCommand(install):
                     "(https://www.rust-lang.org/tools/install) and try again"
                 )
             subprocess.check_call(
-                ["cargo", "rustc", "--bin", "maturin", "--", "-C", "link-arg=-s"]
-            )
+                 ['cargo', 'install', 'maturin', '--no-default-features', '--features=auditwheel,log,human-panic'])
             source = os.path.join(source_dir, "target", "debug", executable_name)
+            source = os.path.join(os.getenv('VIRTUAL_ENV'), 'bin', 'maturin')
+
         # run this after trying to build with cargo (as otherwise this leaves
         # venv in a bad state: https://github.com/benfred/py-spy/issues/69)
         install.run(self)

